File: 10 - Python/Exercícios/04 - Vetores/Exercício 01.py
#Exercício - Negativos

N = int(input("Quantos números você vai digitar? "))

vet = [0] * N

for i in range(N):

    vet[i] = int(input("Digite um número: "))

print("")
print("Números negativos:")

for i in range(N):
    
    if vet[i] < 0:
        print(vet[i])

File: 07 - C++/Exercícios/04 - Vetores/Exercício 01.cpp
//Exercício - Negativos

#include <iostream>

using namespace std;

int main(){

    int i, N;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    int vet[N];

    for(i = 0; i < N; i++){

        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    cout << "\nNumeros negativos:" << endl;

    for(i = 0; i < N; i++){

        if(vet[i] < 0){
            cout << vet[i] << endl;
        }
    }

    return 0;

}